include/mesos/mesos.proto

```diff
@@ -827,6 +827,18 @@ message ExecutorInfo {
  * in the same zone, in different zones in the same region, or in
  * different regions. Note that all masters in a given Mesos cluster
  * must be in the same region.
+ *
+ * Complex deployments may have additional levels of hierarchy: for example,
+ * multiple racks might be grouped together into "halls" and multiple DCs in
+ * the same geographical vicinity might be grouped together. As a convention,
+ * the recommended way to represent additional levels of hierarchy is via dot-
```

Comment:
We don't need to recommend "dot" right? It could be any delimiter that they choose to have? As far as Mesos is concerned, it doesn't matter.

Reply:
I realize that Mesos doesn't care. The goal of having a convention was to promote compatibility across frameworks and clusters that can handle additional levels of hierarchy. Consistency suffers w/o convention. This was the justification for adding the same commentary to the design doc, and it's just as relevant (and much easier to find) here.

Reply:
@vinodkone did you have any other thoughts on this matter?